tests/swarm-deploy/conftest.py

```diff
@@ -44,8 +45,14 @@ def prepare_all_services(
     return services
 
 
+@pytest.fixture(scope="module")
+def create_db_on_start(devel_environ: Dict[str, str]):
+    devel_environ["WEBSERVER_DB_INITTABLES"] = "1"
```

Comment:
I will change this in another pR by migration routings ... but it is ok for now :-)